packages/dds/tree/src/test/simple-tree/api/getJsonSchema.spec.ts

```diff
@@ -144,15 +148,14 @@ describe("getJsonSchema", () => {
 					_treeNodeSchemaKind: NodeKind.Leaf,
 				},
 			},
-			$ref: '#/$defs/test.Array<["com.fluidframework.leaf.string"]>',
+			$ref: "#/$defs/test.array",
 		};
 		assert.deepEqual(actual, expected);
 
 		// Verify that the generated schema is valid.
 		const validator = getJsonValidator(actual);
 
 		// Verify expected data validation behavior.
-		validator(hydrate(Schema, ["Hello", "world"]), true);
```

Comment:
Why was this removed?

Reply:
Class-based Array nodes do not satisfy AJV's array validation. This is known, and supposedly by design. I think there needs to be more discussion about this, since it is a usability issue and a point of potential confusion (switching from structural schema to class schema changes whether or not the resulting array nodes are treated as arrays - very confusing).

I added comments to explain the status.